core/src/main/java/org/mule/runtime/core/api/MuleEvent.java

```diff
@@ -6,6 +6,11 @@
  */
 package org.mule.runtime.core.api;
 
+import java.net.URI;
```

Comment:
Switching import order create unnecessary conflicts.  The importOrder file that most of team use puts Mule imports first, then java, then others.